README.md

```diff
@@ -19,5 +19,6 @@ Syntax:
 |[Aarya Chopkar](https://twitter.com/ChopkarAarya)| Happy birthday. Keep inspiring, continue your amazing work.|
 |[Aditya Joshi](https://twitter.com/aditya__exe) | Happy Birthday Eddie! Loads of love❤️|
 |[Pushkaraj Kulkarni](https://twitter.com/pushkaraj2007)|Happy Birthday Eddie! Have a great year ahead, and Thanks for motivating us to start open source journey.|
-|||
-
+|[Shubhangi](https://twitter.com/mshubhangi12)| Happy Birthday, Eddie! On your birthday, On your birthday, I wish you a lifetime of good health so that you can continue to inspire us and guide us during tumultuous times!! :heart_eyes: :balloon: :confetti_ball: :tada::sparkles:
+|[Krupali Trivedi](https://twitter.com/chai_really)|Happy Birthday Eddie! Thank you for the awesome community. Wishing a happy happy years ahead!|
+|[Shubhangi](https://twitter.com/mshubhangi12)| Happy Birthday, Eddie! On your birthday, On your birthday, I wish you a lifetime of good health so that you can continue to inspire us and guide us during tumultuous times!! :heart_eyes: :balloon: :confetti_ball: :tada::sparkles:
```

Comment:
```suggestion
|[Shubhangi](https://twitter.com/mshubhangi12)| Happy Birthday, Eddie! On your birthday, On your birthday, I wish you a lifetime of good health so that you can continue to inspire us and guide us during tumultuous times!! :heart_eyes: :balloon: :confetti_ball: :tada::sparkles:
|||
```